src/Directory.Build.props

```diff
@@ -0,0 +1,8 @@
+<Project>
+
+  <PropertyGroup>
+    <LangVersion>latest</LangVersion>
```

Comment:
This isn't a clean up, but rather it allows the latest C# minor and instead of the latest major, but I think it's OK to be part of this PR.